torch/functional.py

```diff
@@ -1076,7 +1077,7 @@ def atleast_3d(*tensors):
         >>> torch.atleast_3d(y)
         tensor([[[-0.8079],
                 [ 0.7460]],
-
+                <BLANKLINE>
```

Comment:
doctests need this to pass

Reply:
There are zero other instances of `BLANKLINE`, so I guess that makes this the only 3-D array in an example. Hence not worth cleaning this up with better doctest machinery I think.

A bit odd though that `doctest` failures don't turn into actual failures, that should be separate from `-WT` right?